crates/oxc_linter/src/snapshots/no_access_key.snap

```diff
@@ -0,0 +1,93 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: no_access_key
+---
+  ⚠ eslint-plugin-jsx-a11y(no-access-key): No access key attribute allowed. Inconsistencies between keyboard shortcuts and keyboard commands used by screenreaders and keyboard-only users create
+  │ a11y complications.
+   ╭─[no_access_key.tsx:1:1]
+ 1 │ <div accesskey="h" />
```

Comment:
shall we change this to report on the `accesskey` prop instead?

Reply:
@camc314 

I fixed this. 
Thank you for your review!

a76643009a0ae952b3e0402eef71d46e8ad24b44